Documentation/Introduction/Index.rst

```diff
@@ -31,8 +31,8 @@ classes (add new methods, modify the method bodies, add comments and annotations
 extension builder preserves your changes, even if you then rename models or properties in
 the modeler.
 
-The ExtensionBuilder is NOT intended to do TYPO3 CMS specific stuff, like extending pages or custom content elements,
-as the the old kickstarter. It focuses on extensions development in a sense of `Domain Driven Design <https://de.wikipedia.org/wiki/Domain-Driven_Design>`_
+The Extension Builder is NOT intended to do TYPO3 CMS specific stuff, like extending pages or custom content elements,
```

Comment:
I'm not sure about this as ExtensionBuilder is the product name. Otherwise we should write `extension builder` in lowercase

Reply:
Extension Builder can be in upper case because it is the name of this extension. See Extension Manager is also written like this.